readme.md

```diff
@@ -37,6 +38,29 @@ This project is a Discord bot built with `discord.js`. The bot includes various
    - **DISCORD_CLIENT_ID**: Your bot's client ID.
    - **DISCORD_GUILD_ID**: The ID of the server (guild) where the commands will be deployed.
 
+To obtain each of these environment variables, you need to have `Developer Mode` enabled in Discord (if it is not already).
+
+Open Discord to enable developer mode and click on the gear icon (User Settings) at the bottom left. Go to `App Settings` and select the `Advanced` option, then look for the `Developer Mode` option and toggle it on. For more details on how to create a Discord bot, you can visit the official Discord guide.
+
+Once `Developer Mode` is enabled, you can follow these steps to obtain Environment Variables:
+
+### 1. Obtain **DISCORD_TOKEN**
+- Go to the [Discord Developer Portal](https://discord.com/developers/applications) and log in.
+- Create a new application by clicking on “New Application”.
+- Name your application and click "Create".
+- In the "Bot" section, click on "Add Bot" and confirm.
+- Copy the bot token that is generated. This is your **DISCORD_TOKEN**.
+
+### 2. Obtain **DISCORD_CLIENT_ID**
+- In the Discord Portal (not Discord Developer Portal), select or click on your user avatar on the bottom part when located on the `@me channel`.
+- You will find the option to copy the `Client ID` in the menu called `Copy User ID`. This is your **DISCORD_CLIENT_ID**.
+
+### 3. Obtain **DISCORD_GUILD_ID**
+- Open Discord and go to the server from which you want to obtain the ID.
+- Right-click on the server name and select `Copy Server ID`. This is your **DISCORD_GUILD_ID**.
```

Comment:
I suggest a link to an official tutorial as complement, or leave images as references of where to find the ID visually.

Reply:
The link to an official tutorial is there:
>![image](https://github.com/user-attachments/assets/3cb3d442-9f99-43dc-bbd6-795c139281c7)